### PR TITLE
Improve / add some German translations

### PR DESCRIPTION
**Pull request by realulim**
No code changes, just some German localisations

### CODE REVIEW

File: plugins/emoji/trumbowyg.emoji.js [hunk 1]
```diff
@@ -1275,6 +1275,9 @@
             da: {
                 emoji: 'Tilføj et humørikon'
             },
+	    de: {
+		emoji: 'Emoticon einfügen'
+	    },
```
Comment (Alex-D) [inline plugins/emoji/trumbowyg.emoji.js]: Please use correct indentation

File: plugins/upload/trumbowyg.upload.js [hunk 2]
```diff
@@ -58,6 +58,11 @@
                 file: 'Fil',
                 uploadError: 'Fejl'
             },
+	    de: {
+		upload: 'Hochladen',
+		file: 'Datei',
+		uploadError: 'Fehler'
+	    },
```
Comment (Alex-D) [inline plugins/upload/trumbowyg.upload.js]: indent

Comment (realulim) [reply plugins/upload/trumbowyg.upload.js]: My IDE shows both versions alike, but here's for you vi guys :)

File: plugins/resizimg/trumbowyg.resizimg.js [hunk 3]
```diff
@@ -38,7 +38,7 @@
                                     return false;
                                 },
                                 onDragEnd: function () {
-                                    trumbowyg.$c.trigger('tbwchange');
+                                    trumbowyg.$c.trumbowyg('execCmd', {cmd: 'syncCode'});
```
Comment (Alex-D) [inline plugins/resizimg/trumbowyg.resizimg.js]: From here, you should call `trumbowyg.syncCode()` since we have direct access to the internal API :)

Comment (realulim) [reply plugins/resizimg/trumbowyg.resizimg.js]: Sure, no prob.

**Pull request merged**